### ISSUE TITLE
why torch.optim.lr_scheduler.ReduceLROnPlateau is not implemented?

### ISSUE THREAD

**Issue by RaymondByc** [open-mmlab/mmcv#991]
torch.optim.lr_scheduler.ReduceLROnPlateau is a efficient lr scheduler which could reduce the  training time dymatically, I think.

why it is not  implemented? 

what insight behind it?

**Comment by zhouzaida** [comment]
hi @RaymondByc , ReduceLROnPlateau is not used by downstream repos  so we currently do not support it but in our plan. Are you willing to implement it?

**Comment by RaymondByc** [comment]
Sorry, I do not have enough time to challenge it. I am full-time engineer, and I am occupied by a bunch of works.

**Comment by zhouzaida** [comment]
refer to #1033
